data/data/gcp/control/main.tf

```diff
@@ -0,0 +1,40 @@
+resource "google_compute_instance" "control" {
+  name         = "${var.cluster_id}-control-${count.index}"
```

Comment:
this name must match the machine object's name for adoption... 
https://github.com/openshift/installer/blob/21570f8a4ab99a9de874397e6fbe42ba55dbd29a/pkg/asset/machines/gcp/machines.go#L48

``clusterid-master-idx`